src/main/java/com/snowflake/kafka/connector/internal/SnowflakeConnectionServiceV1.java

```diff
@@ -577,16 +578,17 @@ public boolean isStageCompatible(final String stageName) {
     checkConnection();
     InternalUtils.assertNotEmpty("stageName", stageName);
     if (!stageExist(stageName)) {
-      LOGGER.debug("stage {} doesn't exists", stageName);
+      LOGGER.info("stage {} doesn't exists", stageName);
       return false;
     }
     List<String> files = listStage(stageName, "");
     for (String name : files) {
       if (!FileNameUtils.verifyFileName(name)) {
-        LOGGER.debug("file name {} in stage {} is not valid", name, stageName);
+        LOGGER.info("file name {} in stage {} is not valid", name, stageName);
```

Comment:
I think we can leave it on debug.

Reply:
I would like to give it a try on the INFO level. 

Customer reported an error that occasionally some files are left on a stage and it causes connector to restart. Getting file names could greatly improve our debugging capabilities and DEBUG level is unfortunately not available.